pkg/virtualKubelet/apiReflection/reflectors/outgoing/secrets.go

```diff
@@ -200,14 +212,11 @@ func (r *SecretsReflector) PreDelete(obj interface{}) interface{} {
 }
 
 func (r *SecretsReflector) isAllowed(obj interface{}) bool {
-	sec, ok := obj.(*corev1.Secret)
+	_, ok := obj.(*corev1.Secret)
```

Comment:
Since we do not need the blacklisting mechanism anymore, you can delete this function. You should also delete the field  assignation [here](https://github.com/liqotech/liqo/blob/f5cf34fc99c77ff19d7cf0a9e37c615f0003a319/pkg/virtualKubelet/apiReflection/reflectors/outgoing/secrets.go#L24)